### ISSUE TITLE
Encoding the Loss Function

### ISSUE THREAD

**Issue by ttorku** [lululxvi/deepxde#189]
How do we encode the physics of the model into the loss function. I am familiar with PINN from your group. But in Deepxde, I am not familiar with how to encode the loss function. For example consider the a simple logistic growth model 
**dc/dt =r*c(t)(1-\dfrac{c(t)}{k}) where r, k are the parameters of the model**. If I choose to define my loss function as follows to capture the dynamics
**loss =\dfrac{1}{N}\sum_{i=1}^N (\hat{c}-c)^2 +M|ic| +residual** where ic is the difference in the initial condition of c and the initial of c_pred , M is the size of  c, N is the length of the training data points, residual is e =dc/dt -r*c(t)(1-\dfrac{c(t)}{k})
How do I do achieve this in your framework?? Hoping to hear from anyone soon, especially @lululxvi . Thanks in advance.

**Comment by lululxvi** [comment]
There are many examples in the repo. Check the examples folder.
